hnn_core/tests/test_network.py

```diff
@@ -796,12 +824,12 @@ def test_tonic_biases():
 
     # new API
     net = Network(params)
-    net.add_tonic_bias(cell_type='L2_pyramidal', amplitude=1.0)
```

Comment:
While this API is going to be deprecated, we should have at least one test to make sure it still functions until we completely deprecate it

Reply:
also we should check that the deprecation warning is raised. Leave a clear comment to indicate which part of the test is for deprecated functionality so it can be deleted.

Reply:
I added test to check deprecation warnings